lib/game.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+class Game
+  attr_reader :scores, :frame
+
+  TOTAL_FRAMES = 10
+  MAX_POINTS = 10
+
+  def initialize
+    @frame = 0
+    @scores = []
+  end
+
+  def roll(pins)
+    raise 'Your game is now complete' if game_over?
+    @scores << pins
```

Comment:
Nice way to tell the user they've finished and in irb it stops any further adding to the score.
before you raise the error could you call the score method so the player knows they're total - at the end it should then read their total score and say Your game is now complete?